portal/src/main/webapp/js/src/patient-view/util/ClinicalAttributesUtil.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+/**
+ * Functions for dealing with clinical attributes.
+ */
+window.ClinicalAttributesUtil = (function() {
+  /**
+   * Clean clinical attributes. Useful for rounding numbers, or other types of
+   * data cleanup steps. Probably differs per institution.
+   * @param {object} clinicalData - key/value pairs of clinical data
+   */
+  function clean(clinicalData) {
+    // Shallow Copy clinicalData
+    var cleanClinicalData = $.extend({}, clinicalData);
+    var NULL_VALUES = [
```

Comment:
I agree we should keep this at the java layer (actually I would prefer it to be even before that, in the T step of the ETL...and we are making progress with this...but not there yet, and apparently there is still a need to "harmonize on the fly" some old data that is already in the DB). See also PR #1397 where new "NULL_VALUES" were added to the java enum and the validator list.